docs/governance/README.md

```diff
@@ -0,0 +1,132 @@
+# .NET Standard governance
+
+The .NET Standard represents the set of APIs that all .NET implementations have
+to provide. Over time, this list has to grow but it's critical that the set
+remains implementable by all current and future .NET implementations.
+
+To ensure that's the case, there is a formal process for proposing and accepting
+APIs into the .NET Standard. This document outlines the process.
+
+## Review board
+
+Today, we have several different .NET implementations:
+
+* .NET Core
+* .NET Framework
+* Mono
+* Unity
+* Xamarin
+
+Some of them have drastically different constraints, based on their runtime
+environment (for instance, Xamarin for iOS cannot perform runtime code
+generation, based on Apple's app store policy).
+
+To ensure we don't end up adding large chunks of API surface that cannot be
+implemented, we have a [review board](board.md) that has to sign-off on API
+additions to the .NET Standard. The board comprises representatives from the
+following groups:
+
+  * **.NET platform**. The rationale here is that most, if not all, of the APIs
+    that are part of .NET Standard are implemented and evolved by the .NET
+    platform team.
+  * **Xamarin & Mono**. While Mono mostly copies code from the .NET Framework &
+    .NET Core base class libraries, changes and extensions can impact their
+    ability to support the .NET Standard. Thus, we need to coordinate any
+    changes with Mono.
+  * **Unity**. Same rationale as for Xamarin & Mono.
+  * **.NET Foundation**. A set of people selected by the .NET Foundation that
+    represent the interests of the .NET community at large, which also includes
+    poor consumers of the .NET Standard.
```

Comment:
Is this really meant to be "poor consumers"?

Reply:
I meant "pure consumers" but apparently 

![image](https://user-images.githubusercontent.com/5169960/42058455-4cfde766-7ad5-11e8-90d9-60142656b62c.png)